protocol/pom.xml

```diff
@@ -69,6 +69,10 @@
             <groupId>org.jboss.remoting</groupId>
             <artifactId>jboss-remoting</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I've checked and `org.jboss.as.protocol` module already had a dependency on `org.wildfly.common`: https://github.com/wildfly/wildfly-core/blob/150c652e5b2e916751afada7cddaf0367765961d/core-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/as/protocol/main/module.xml#L40